scripts/__tests.yaml

```diff
@@ -0,0 +1,18 @@
+# copy this file and name it tests.yaml
```

Comment:
Add here
"create your own test sets using these commands - not all of them are necessary"